app/models/concerns/foreman_ansible/host_managed_extensions.rb

```diff
@@ -14,5 +14,24 @@ def inherited_ansible_roles
         hostgroup.all_ansible_roles
       end
     end
+
+    def ansible_proxies(authorized = true)
```

Comment:
This doesn't belong in Host::Managed, move it to a service or similar. It just needs 'location/organization' to work.